DOCS-4164: Update IPMI password requirements (#1588)
truenas/documentation: content/SCALE/SCALEUIReference/Network/IPMIScreens.md

| **IPv4 Netmask** | Enter the subnet mask of the IPv4 address. |
| **IPv4 Default Gateway** | Enter the default gateway of the IPv4 connection. |
| **VLAN ID** | Enter the VLAN identifier if the IPMI out-of-band management interface is not on the same VLAN as management networking. |

> | **Password** | Enter the password to use to connect to the IPMI interface from a web browser. Must include at least one upper, lower case letter, one digit, one special character (punctuation i.e., ! # $ %, etc.) and be 8-16 characters in length. |

Reviewer: ```suggestion
| **Password** | Enter a password for connecting to the IPMI interface from a web browser. The password must include at least one upper case letter, one lower case letter, one digit, and one special character (punctuation, e.g. ! # $ %, etc.). It must also be 8-16 characters long. |
```
